plugins/woocommerce/tests/performance/requests/shopper/my-account-orders.js

```diff
@@ -98,9 +98,16 @@ export function myAccountOrders() {
 			}
 		);
 
+		check( my_account_order_id, {
+			'order ID is not undefined': () => {
+				console.log( `order id: ${ my_account_order_id }` );
```

Comment:
Generally we want to avoid console.log in case we generate large amounts of logs during a load test, but I think given the other changes this will hopefully not be triggered much/ever. Although maybe that means we can remove it?

Reply:
+1 on removing this. I believe it was left in my code sample for debugging.

Reply:
Done in a new commit